### DISCUSSION TITLE
Character Persona not working?

### DISCUSSION THREAD

**Discussion by patrickmros** [oobabooga/text-generation-webui#361] [Q&A]
If I describe a person in the context field and ask the character in the chat for some characteristics of himself i get wrong answers. I wrote "Lily is a 30 years old woman" and get as an answer "I'm 21". Did i do something wrong?

My context begins like this: 

Lily's Persona: Lily is 30 years old and likes reading books (...)

**Comment by VldmrB** [comment]
Not sure... All LLMs can make stuff up at times. But the `Lily's Persona: ` part seems unnecessary, and could possibly be contributing. Though I don't know the model you're using.

**Reply by patrickmros** [reply]
Hm… I did not invent the persona part. It‘s generated automatically when using [oobaboogas character creator ](https://oobabooga.github.io/character-creator.html) and if i remember correctly it‘s formated like this in the example character. I have now tried the tavernAI character cards, too. Same result. Don‘t know if i do something wrong or if the models are just not good enough….

I tried pygmalion-6b, llama-7b and Lama-13b.

**Reply by VldmrB** [reply]
Yea, maybe, I'm not that familiar with it. Using Llama-13b/30b, they make silly mistakes all the time, but I rarely ask questions with very specific answers.
